Information/CenterOfMass.FCMacro

```diff
@@ -1417,7 +1468,7 @@ if __name__ == '__main__':
     elif not app.activeDocument():
         error_message('Open a document first.')
     else:
-        print('Loading ' + __Name__ + ' ' + __Version__ + ' ...')
+        app.Console.PrintMessage(f'Loading {__Name__} {__Version__} ...' + '\n')
```

Comment:
Why not `app.Console.PrintMessage(f'Loading {__Name__} {__Version__} ...\n')`?

Reply:
Originally to improve readability by avoiding `word\nword` and allowing line breaks in the code, but in this case of course totally unnecessary.